frontend/src/features/moti/mapSideBar/containers/MotiInventoryContainer.tsx

```diff
@@ -152,16 +152,17 @@ export const MotiInventoryContainer: React.FunctionComponent = () => {
         setLoading(true);
         const parcelLayerResponse = await getParcelLayerResponse(latLng, pid);
         const properties = getIn(parcelLayerResponse, 'features.0.properties');
-        if (!properties?.PID) {
+        pid = pid ?? properties?.PID;
+        if (pid && !(await api.isPidAvailable(undefined, pid)).available) {
```

Comment:
prioritize displaying duplicate errors over any other error type (for cases where an invalid pid has already been added to PIMS).